src/org/labkey/test/TestProperties.java

```diff
@@ -84,6 +84,16 @@ public static boolean isTestRunningOnTeamCity()
         return !undefinedBuildTypeProperty.equals(System.getProperty(buildTypeProperty, undefinedBuildTypeProperty));
     }
 
+    public static boolean isServerRemote()
+    {
+        return "true".equals(System.getProperty("labkey.isRemote", "true"));
+    }
+
+    public static boolean isIgnoreMissingModules()
+    {
+        return "true".equals(System.getProperty("webtest.ignoreMissingModules", "true"));
```

Comment:
I would not return true by default for these two system properties labkey.isRemote & webtest.ignoreMissingModules.

You will impact individuals local runs in ways they may not expect if they do not have these properties set.

Reply:
okay.  I will update these to default to 'false'.